utils/my_config_file.py

```diff
@@ -136,6 +137,21 @@ class Charts(Enum):
         id="id_psy_to_chart",
         note_chart="In this psychrometric chart the abscissa is the operative temperature and for each point dry-bulb temperature equals mean radiant temperature (DBT = MRT). The comfort zone represents the combination of conditions with the same DBT and MRT for which the PMV is between -0.5 and +0.5, according to the standard.",
     )
+    as_psychrometric: ChartsInfo = ChartsInfo(
```

Comment:
there is no air speed psychrometric chart.

Reply:
![image](https://github.com/user-attachments/assets/6fe2d6e8-1d08-4f06-8e5f-7acef333f146)
it's follow the previous, beacause of back- end need all those charts slots

Reply:
this is not a psychrometric chart

Reply:
wind_temp_chart

Reply:
I change the name like this
![image](https://github.com/user-attachments/assets/a18c8abc-dc29-4fa5-acd3-d5b687816207)